### DISCUSSION TITLE
How to improve Loading speed in Next.js?

### DISCUSSION THREAD

**Discussion by coincatcher22** [community/community#67824] [Other Feature Feedback, Questions, & Ideas]
Hi there,

Normally, in Next.js the loading speed is slow.

How to fix it as possible?

**Comment by ibtissammassa** [comment]
Hey there,

I totally get where you're coming from. I faced a similar issue with Next.js and spent some time digging into ways to improve loading speed. Here are a few things that could potentially help:

So, one of the initial steps I took was analyzing the errors flagged by _**Lighthouse**_. That really shed light on areas needing improvement, allowing me to address them directly.

_**Upgrading Next.js**_ was a game-changer for me. It often resolves a bunch of underlying issues and comes with performance enhancements.

I also experimented with` npm install sharp` – this significantly sped up image processing for me. Remember, it's crucial to ensure it's installed on the server as well for optimal results.

Tweaking the caching settings in next.config.js might help too. Setting _**minimumCacheTTL**_ can notably improve caching and subsequently, loading speed.

```
// Inside next.config.js
module.exports = {
  images: {
    minimumCacheTTL: 31536000,
  },
}
```

Another thing that made a difference for me was tweaking image loading. I prioritized the most crucial images by using the `priority={true}` prop. This basically bypasses lazy loading for those images, resulting in faster rendering.

Lastly, implementing `placeholder="blur"` for images made a noticeable difference in perceived performance. It's a small addition but can significantly enhance user experience during loading.

You don't have to try all of these—some are just results of my research when I encountered the same issue. They're suggestions that might help, but every project's different. Feel free to pick and choose, and if you need more details on any specific suggestion, I'm here to assist
